test/user.test.js

```diff
@@ -205,6 +205,60 @@ describe('User', function() {
       });
     });
 
+    describe('Password validation with Maxlength', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIZ';
+
+      it('invalidates created account when password is greater than Maxlength', function(done) {
+        try {
+          User.create({ email: 'b@c.com', password: pass73 }, function(err) {
+            if (err) return done(err);
+            assert(err);
+            assert.equal(err.statusCode, 422);
+            assert(false, 'Error should have been thrown');
+          });
+        } catch (e) {
+          done();
+        }
+      });
+
+      it('validates created account when password is at Maxlength', function(done) {
+        User.create({ email: 'b@c.com', password: pass72 }, function(err, user) {
+          if (err) return done(err);
+          assert(user.id);
+          assert(!err);
+
+          done();
+        });
+      });
+
+      it('validates login when password is at Maxlength', function(done) {
+        User.create({ email: 'b@c.com', password: pass72 }, function(err) {
+          User.login({ email: 'b@c.com', password: pass72 }, function(err, accessToken) {
+            if (err) return done(err);
+            assertGoodToken(accessToken);
+            assert(accessToken.id);
+            assert(!err);
+
+            done();
+          });
+        });
+      });
+
+      it('invalidates login when password is at Maxlength but wrong', function(done) {
```

Comment:
I think in this test, you need to take password MAXLEN long and add few more characters to it.

As this test is written now, it passes even without the changes you made in `common/models/user.js`. Have you seen this test fail?